From: Cole Robinson <crobinso@redhat.com>
Date: Mon, 10 Aug 2015 13:01:04 -0400
Subject: [PATCH virt-manager] osdict: Fix unix alias

Needs to point to proper libosinfo name freebsd9.0

(cherry picked from commit 55b43dcbb81305e1cad6d157efd4d229a0b26741)
---
 virtinst/osdict.py | 7 ++-----
 1 file changed, 2 insertions(+), 5 deletions(-)

diff --git a/virtinst/osdict.py b/virtinst/osdict.py
index 76c2260..a6ae3ac 100644
--- a/virtinst/osdict.py
+++ b/virtinst/osdict.py
@@ -174,7 +174,7 @@ class _OSDB(object):
         "linux" : "generic",
         "windows" : "winxp",
         "solaris" : "solaris10",
-        "unix": "freebsd9",
+        "unix": "freebsd9.0",
         "other": "generic",
     }
 
@@ -221,10 +221,7 @@ class _OSDB(object):
 
     def lookup_os(self, key):
         key = self._aliases.get(key) or key
-        ret = self._all_variants.get(key)
-        if ret is None:
-            return None
-        return ret
+        return self._all_variants.get(key)
 
     def lookup_os_by_media(self, location):
         media = libosinfo.Media.create_from_location(location, None)
